packages/lesswrong/server/notificationTypesServer.tsx

```diff
@@ -417,6 +418,40 @@ export const NewGroupOrganizerNotification = serverRegisterNotificationType({
   },
 });
 
+export const NewCommentOnDraftNotification = serverRegisterNotificationType({
```

Comment:
Emails users when someone comments on a draft post in collaborative editing mode